translations/strings.yaml

```diff
@@ -178,7 +178,6 @@ inAppSupportWorkflow:
   supportErrorButton: Try Again
```

Comment:
Again, VSCode just formatted this file on save. If that is an issue I am happy to revert the changes.